install/alerting/alertmanager.tmpl

```diff
@@ -0,0 +1,38 @@
+global:
+  # ResolveTimeout is the default value used by alertmanager if the alert does
+  # not include EndsAt, after this time passes it can declare the alert as resolved if it has not been updated.
+  # This has no impact on alerts from Prometheus, as they always include EndsAt.
+  # default = 5m
+  resolve_timeout: 5m
+
+route:
+  # The labels by which incoming alerts are grouped together.
+  group_by: ["alertname"]
+  # How long to initially wait to send a notification for a group
+  # of alerts. Allows to wait for an inhibiting alert to arrive or collect
+  # more initial alerts for the same group.
+  group_wait: 30s
+  # How long to wait before sending a notification about new alerts that
+  # are added to a group of alerts for which an initial notification has
+  # already been sent. (Usually ~5m or more.)
+  group_interval: 5m
+  # How long to wait before sending a notification again if it has already been sent successfully for an alert.
+  repeat_interval: 4h
+  # The notification destination
+  receiver: "node_operator"
```

Comment:
I considered that. I think all of these are the defaults and I put them here mostly for clarity and awareness of the capabilities for future maintenance. 

IMHO since this could always be added later, worth leaving it as is for now and waiting to see if users actually want such fine-grained control. If they don't, its less docs, questions and maintenance. As always, I'm just sharing thoughts. I'm open.